### PR TITLE
feat: adjust dynamic-react css vars and scss build process

### PR DESCRIPTION
**Pull request by abe-modyo**

### CODE REVIEW

File: libraries/dynamic-ui/src/style/abstracts/variables/_badges.scss [hunk 1]
```diff
@@ -3,13 +3,13 @@
 // scss-docs-start badge-variables
 $badge-font-size: $small-font-size !default;
 $badge-font-weight: $font-weight-normal !default;
-$badge-color: $color-contrast-dark !default;
+$badge-color: var(--#{$prefix}gray-900) !default;
 $badge-padding-y: $spacer-1 !default;
 $badge-padding-x: $spacer-2 !default;
```
Comment (johannrp27) [inline libraries/dynamic-ui/src/style/abstracts/variables/_badges.scss:8]: .

File: libraries/dynamic-ui/src/style/abstracts/variables/_dropdowns.scss [hunk 2]
```diff
@@ -30,7 +30,7 @@ $dropdown-link-disabled-color: var(--#{$prefix}tertiary-color) !default;
 $dropdown-item-padding-y: $spacer * .25 !default;
```
Comment (johannrp27) [inline libraries/dynamic-ui/src/style/abstracts/variables/_dropdowns.scss:30]: .

File: libraries/dynamic-ui/src/style/abstracts/variables/_list-group.scss [hunk 3]
```diff
@@ -15,29 +15,29 @@ $list-group-item-bg-scale: 0% !default;
...
 // custom
 $list-group-item-border-radius: 0 !default;
 
-$list-group-item-bg-light: $light !default;
+$list-group-item-bg-light: var(--#{$prefix}light) !default;
 $list-group-item-picture-width: 2.25rem !default;
 $list-group-item-picture-height: 2.25rem !default;
 $list-group-item-picture-border-radius: $border-radius-sm !default;
 
 $list-group-item-content-padding-y: $spacer-3 !default;
```
Comment (johannrp27) [inline libraries/dynamic-ui/src/style/abstracts/variables/_list-group.scss:40]: .

File: libraries/dynamic-ui/src/style/abstracts/variables/_typography.scss [hunk 4]
```diff
@@ -94,7 +94,7 @@ $initialism-font-size: $small-font-size !default;
 
 $blockquote-margin-y: $spacer !default;
 $blockquote-font-size: $font-size-base * 1.25 !default;
-$blockquote-footer-color: $gray-600 !default;
+$blockquote-footer-color: var(--#{$prefix}gray-600) !default;
 $blockquote-footer-font-size: $small-font-size !default;
 
 $hr-margin-y: $spacer !default;
```
Comment (johannrp27) [inline libraries/dynamic-ui/src/style/abstracts/variables/_typography.scss:100]: .

File: libraries/dynamic-ui/src/style/abstracts/variables/_typography.scss [hunk 5]
```diff
@@ -94,7 +94,7 @@ $initialism-font-size: $small-font-size !default;
 
 $blockquote-margin-y: $spacer !default;
```
Comment (johannrp27) [inline libraries/dynamic-ui/src/style/abstracts/variables/_typography.scss:95]: .

File: libraries/dynamic-ui/src/style/base/_tables.scss [hunk 6]
```diff
@@ -0,0 +1,171 @@
...
+  --#{$prefix}table-accent-bg: #{$table-accent-bg};
+  --#{$prefix}table-striped-color: #{$table-striped-color};
+  --#{$prefix}table-striped-bg: #{$table-striped-bg};
+  --#{$prefix}table-active-color: #{$table-active-color};
+  --#{$prefix}table-active-bg: #{$table-active-bg};
+  --#{$prefix}table-hover-color: #{$table-hover-color};
+  --#{$prefix}table-hover-bg: #{$table-hover-bg};
+
+  width: 100%;
+  margin-bottom: $spacer;
```
Comment (johannrp27) [inline libraries/dynamic-ui/src/style/base/_tables.scss:24]: .

**Review by johannrp27 — APPROVED** [review]

**Review by efbarong — APPROVED** [review]

**Review by johannrp27 — APPROVED** [review]

**Review by efbarong — APPROVED** [review]

**Pull request merged**